wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/ProfileActionsFactory.swift

```diff
@@ -190,7 +191,7 @@ final class ProfileActionsFactory {
                 actions.append(.cancelConnectionRequest)
             } else if (user.isConnected && !user.hasEmptyName) || isOnSameTeam {
                 actions.append(.openOneToOne)
-            } else if user.canBeConnected && !user.isPendingApprovalBySelfUser {
+            } else if user.canBeConnected && !user.isPendingApprovalBySelfUser && isUserFormClassifiedBackend(user: user) {
```

Comment:
Why do we need this check?

Reply:
to hide "connect" button if backends not federating